backend/src/dao/AdminsDao.ts

```diff
@@ -0,0 +1,8 @@
+import { adminCollection } from '../firebase';
+
+export default class Adminsdao {
```

Comment:
Camel case for consistency
```suggestion
export default class AdminsDao {
```